README.md

```diff
@@ -18,4 +18,8 @@ POST [project_url]/api/shorturl/new - body (urlencoded) :  url=https://www.googl
 
 #### Will redirect to:
 
-https://www.freecodecamp.org/forum/
```

Comment:
Your explanation is really good, but not necessary for the README. This might be better as general information on the forum for the backend projects.